templates/default/pom.xml

```diff
@@ -87,6 +87,12 @@
             <version>1.1.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I think we should depend on  
org.microshed: microshed-testing-liberty
instead.